packages/domain/source/server/todos/todo-list.js

```diff
@@ -1,8 +1,10 @@
 Space.eventSourcing.Aggregate.extend('Todos.TodoList', {
 
-  fields: {
-    name: String,
-    todos: [Todos.TodoItem]
+  fields() {
```

Comment:
Pretty sure we're not going to merge that PR, as this particular API probably won't change